c/pthread-memsafety/list1_false-valid-deref.i

```diff
@@ -1070,6 +1070,8 @@ int main(int argc, char **argv) {
   pthread_t id1, id2;
   struct item *list = ((void *)0);
   pthread_create(&id1, ((void *)0), build, list);
+  pthread_join(id1, ((void *)0));
```

Comment:
This change removes the main part of the concurrent program.
The program should have concurrent access to a list and then dereference some already deleted element.